pkg/bench/get.go

```diff
@@ -292,10 +293,16 @@ func (g *Get) Start(ctx context.Context, wait chan struct{}) (Operations, error)
 				}
 
 				if g.RandomRanges && op.Size > 2 {
-					// Randomize length similar to --obj.randsize
-					size := generator.GetExpRandSize(rng, 0, op.Size-2)
-					start := rng.Int63n(op.Size - size)
-					end := start + size
+					var start, end int64
+					if g.RangeSize == 0 {
+						// Randomize length similar to --obj.randsize
+						size := generator.GetExpRandSize(rng, 0, op.Size-2)
+						start = rng.Int63n(op.Size - size)
+						end = start + size
+					} else {
+						start = g.RangeSize * rng.Int63n(op.Size/g.RangeSize)
```

Comment:
Why not a random start?
```suggestion
						start = rng.Int63n(op.Size - g.RangeSize)
```